conf/airframes/HooperFly/racerpex_hexa_lisa_mx_20.xml

```diff
@@ -47,6 +49,17 @@
     <load name="gps_ubx_ucenter.xml"/>
     <load name="geo_mag.xml"/>
     <load name="air_data.xml"/>
+    <load name="nav_survey_rectangle_rotorcraft.xml">
+      <define name="RECTANGLE_SURVEY_DEFAULT_SWEEP" value="10"/>
+    </load>
+    <load name="nav_survey_poly_rotorcraft.xml"/>
```

Comment:
You are refering to `nav_survey_poly_rotorcraft`, correct? I will comment that module out of each file. Thanks for catching that.